src/entities/TimeRange.svelte

```diff
@@ -11,9 +11,14 @@
     $: {
         _position.x = left, _position.width = width;
     };
+
+    function setClass(node){
+        if(!model.classes) return;
+        node.classList.add(model.classes);
+    }
 </script>
 
-<div class="sg-time-range" class:moving="{resizing}" style="width:{_position.width}px;left:{_position.x}px">
+<div class="sg-time-range" class:moving="{resizing}" style="width:{_position.width}px;left:{_position.x}px" use:setClass>
```

Comment:
I know this is already merged but maybe it would be better to use class="sg-time-range {model.classes ? model.classes.join(' ') : ''}" because the value would then be bound

Reply:
you're absolutely right, I'll do the modification tomorrow!